packages/application-generic/src/health/completed-jobs-metric-queue.health-indicator.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  HealthCheckError,
+  HealthIndicator,
+  HealthIndicatorResult,
+} from '@nestjs/terminus';
+import { Injectable, Logger } from '@nestjs/common';
+
+import { CompletedJobsMetricQueueService } from '../services';
+
+const LOG_CONTEXT = 'CompletedJobsMetricQueueServiceHealthIndicator';
+
+@Injectable()
+export class CompletedJobsMetricQueueServiceHealthIndicator extends HealthIndicator {
```

Comment:
I don't see that this one `CompletedJobsMetricQueueServiceHealthIndicator` and `ActiveJobsMetricQueueServiceHealthIndicator` is used anywhere. Will we use them in the next PR?

Reply:
What do you mean by this?
I already have better uptime cron health attached to this.

Reply:
@Cliftonz 
I see that there is no code that is using these two classes. They are imported in the `packages/application-generic/src/modules/queues.module.ts` but not used.